pkg/sdk/go/users.go

```diff
@@ -188,6 +189,37 @@ func (sdk mfSDK) UpdateUserIdentity(user User, token string) (User, errors.SDKEr
 	return user, nil
 }
 
+func (sdk mfSDK) ResetPasswordRequest(email string) errors.SDKError {
```

Comment:
what is the flow to reset a password, what is the sequence of commands to reset password with cli?

Reply:
set in the current commit